webapp/channels/src/components/youtube_video/youtube_video.test.tsx

```diff
@@ -31,6 +31,7 @@ describe('YoutubeVideo', () => {
                 url: 'linkForThumbnail',
             }],
         },
+        youtubeReferrerPolicy: false,
```

Comment:
Can we add one test (even if it is a snapshot test) with this setting set to true?